<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubMenuClass</class>
 <widget class="QWidget" name="SubMenuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_title" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>800</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);</string>
   </property>
   <widget class="QLabel" name="label_icon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Clock" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>5</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR PL UKai CN</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>99</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>串口通讯</string>
   </property>
  </widget>
  <widget class="QWidget" name="showArea" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>800</width>
     <height>320</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Clock</class>
   <extends>QWidget</extends>
   <header location="global">clock.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
